.github/workflows/test.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  # Trigger the workflow on push or pull request, but only for the develop branch
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ '3.8', '3.9', '3.10-dev' ]
+
+    name: Python ${{ matrix.python-version }} on Ubuntu latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update -qq
+          sudo apt-get install -qq -y gir1.2-gtk-3.0 gobject-introspection libgirepository1.0-dev xvfb
+      - name: Install python dependencies
+        run: |
+          pip install PyGObject
+          pip install pyxdg
+      - name: Configure git
+        run: |
+          git config --global user.email "you@example.com"
+          git config --global user.name "Your Name"
```

Comment:
This is required to fix the following error:

```
ERROR: runTest (tests.versioncontrol.TestGit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/tests/versioncontrol.py", line 339, in runTest
    vcs.commit_version('test 1')
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/zim/plugins/versioncontrol/__init__.py", line 521, in commit_version
    self.commit(None, msg)
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/zim/plugins/versioncontrol/git.py", line 109, in commit
    return self.run(params)
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/zim/plugins/versioncontrol/__init__.py", line 451, in run
    self._app.run(params, self.root)
  File "/home/runner/work/zim-desktop-wiki/zim-desktop-wiki/zim/applications.py", line 269, in run
    raise ApplicationError(argv[0], argv[1:], p.returncode, _decode(stderr))
zim.applications.ApplicationError: Failed to run application: git

git "commit" "-m" "test 1"
returned non-zero exit status 128

Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: empty ident name (for <runner@fv-az190-546.22yfuiovmbsufjzmhpaxkig25b.bx.internal.cloudapp.net>) not allowed
```

See https://github.com/pdecat/zim-desktop-wiki/runs/3707947920?check_suite_focus=true#step:6:1030